### DISCUSSION TITLE
Issues post VM installation and remediation

### DISCUSSION THREAD

**Discussion by gta0gagan** [KharchenkoPM/Interactive-Easy-GPU-PV#1] [General]
So post VM installation I had few issues like:

1. Prasec not working Properly
2. CPU core scheduler not correct - Had to change it to "core" using command
```
#to check what type of scheduler is:
Get-WinEvent -FilterHashTable @{ProviderName="Microsoft-Windows-Hyper-V-Hypervisor"; ID=2} -MaxEvents 1
```
```
#to set to core scheduler
bcdedit.exe /set hypervisorschedulertype Core
```
3. Had to turn on the SVM in system BIOS for  my AMD Zen 4 system. (This one is on me since I am new to Zen platform)

There are still issues like:
1. Game feels choppy with stutters to play even after giving Sufficient RAM (16GiB) and CPU (6 Core, 12 Threads)
2. The game sound not coming properly in Parsec

All in all while this is a good project to experiment, its not ready to actively use and play

**Comment by Jason3DR** [comment]
Are you using dynamic memory, also is that 6 vcores? not sure that is 12 threads.
